components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -56,12 +56,12 @@
     <ul>
       <li class="service-item" *ngFor="let service of services$ | async; let i = index">
         <div class="header">
-          <div class ="fqdn-line"><chef-icon>storage</chef-icon> {{ service.fqdn || "--"}}</div>
-          <div class ="supervisor-line"><b>Supervisor:</b> {{ service.supervisor_id || "--"}}</div>
-          <div class ="site-line"><b>Site:</b> {{ service.site  || "--"}}</div>
+          <div class ="fqdn-line"><chef-icon>storage</chef-icon> <p class="fqdn-data">{{ service.fqdn || "--"}}</p></div>
+          <div class ="supervisor-line"><b>Supervisor:</b> <p class="supervisor-data">{{ service.supervisor_id || "--"}}</p></div>
```

Comment:
i know this is not from you, but can you use strong tags here instead?

Reply:
I have changed to `strong`.